flink-container/pom.xml

```diff
@@ -66,4 +66,64 @@ under the License.
 		</dependency>
 
 	</dependencies>
+
+	<build>
+		<plugins>
+			<!-- Assemble a test jar for EntryClassParserTest. More information on this:
+			 http://stackoverflow.com/questions/1401857/using-maven-to-build-separate-jar-files-for-unit-testing-a-custom-class-loader
+			 -->
+			<plugin>
+				<artifactId>maven-assembly-plugin</artifactId>
```

Comment:
you could simplify this a lof by using the `shade-plugin`; you wouldn't need a separate `assembly.xml`.

Reply:
This is copied from `flink-formats/flink-avro` and `flink-clients` respectively. Happy to get rid of the `assembly.xml` etc. Do you have the shade plugin configuration handy for this?

Reply:
scratch that, we can only use the shade-plugin in/after the package phase, but unit tests are executed before-hand.